test/e2e/kuma_deploy_hybrid_test.go

```diff
@@ -234,4 +243,27 @@ metadata:
 		Expect(err).ToNot(HaveOccurred())
 		Expect(stdout).To(ContainSubstring("HTTP/1.1 200 OK"))
 	})
+
+	It("should sync traffic permissions", func() {
```

Comment:
What is this test about?
Is it about syncing resources? In this case, I think it makes more sense to check what is the state of CPs.
Is it about testing traffic permissions? I think it makes more sense to check it in a test file dedicated to TrafficPermission

Reply:
The primary goal was to check resource synchronization. But we also check how traffic permissions behave cross cluster.